frontend/src/components/resource-links.tsx

```diff
@@ -12,7 +12,7 @@ export const ResourceLinks = () => {
 }
 
 export const HelpLink = () => {
-    const isResearcher = useCurrentUser().isResearcher
+    const isResearcher = useCurrentUser()?.isResearcher
```

Comment:
can `useCurrentUser()` return an empty object while its loading?  that way you wouldn't need to remember to add a `?` everywhere